internal/cli/atlas/backup/restores_start.go

```diff
@@ -63,25 +69,38 @@ func (opts *RestoresStartOpts) Run() error {
 		return err
 	}
 
-	return output.Print(config.Default(), "", r)
+	if opts.isPointInTimeRestore() {
+		return output.Print(config.Default(), pointInTimeTemplate, r)
+	} else if opts.isAutomatedRestore() {
+		return output.Print(config.Default(), automatedTemplate, r)
+	}
+
+	return output.Print(config.Default(), downloadTemplate, r)
```

Comment:
[nit] the only difference is the template, create a function `func (opts*) template() string` or something and just do
```suggestion
	return output.Print(config.Default(), opts.template(), r)
```